ci/jenkins/test.sh

```diff
@@ -868,7 +871,7 @@ function run_conformance_windows {
     export KUBE_TEST_REPO_LIST=${WORKDIR}/repo_list
     if [ "$TESTCASE" == "windows-networkpolicy" ]; then
         # Allow LinuxOnly mark in windows-networkpolicy because Antrea Windows supports NP functions.
-        ginkgo --noColor $E2ETEST_PATH -- --provider=skeleton --ginkgo.focus="$WINDOWS_NETWORKPOLICY_FOCUS" --ginkgo.skip="$WINDOWS_NETWORKPOLICY_SKIP" > windows_conformance_result_no_color.txt || true
+        ginkgo --ginkgo.v=4 --noColor $E2ETEST_PATH -- --provider=skeleton --ginkgo.focus="$WINDOWS_NETWORKPOLICY_FOCUS" --ginkgo.skip="$WINDOWS_NETWORKPOLICY_SKIP" > windows_conformance_result_no_color.txt || true
```

Comment:
Please remove this change after the tests pass.

Reply:
This will be removed as soon as network policy tests pass